tests/test_gui_litellm_catalog_cache.py

```
@@ -0,0 +1,126 @@
+        with tempfile.TemporaryDirectory() as temp_dir:
+            cache = LiteLLMCatalogCache(Path(temp_dir) / "catalog.json")
+
+            self.assertEqual(cache.selected_provider, "")
+            self.assertEqual(cache.providers.values, ())
+            self.assertEqual(cache.models("openai").values, ())
+            self.assertEqual(cache.selected_model("openai"), "")
+
+    def test_catalogs_and_per_provider_selections_survive_reload(self):
+        with tempfile.TemporaryDirectory() as temp_dir:
+            path = Path(temp_dir) / "catalog.json"
+            now = lambda: datetime(2026, 7, 28, 12, 0, tzinfo=timezone.utc)
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `now` as a `def`

(E731)